### DISCUSSION TITLE
Keep single/double quotes

### DISCUSSION THREAD

**Discussion by IanTerzo** [wilsonzlin/minify-html#57] [Q&A] [answered]
For example, `onclick="alert(1)"`  gets minified to `onclick=alert(1)`. How would I go about preventing this?

**Comment by wilsonzlin** [comment, marked as answer]
Currently there is no option to disable this, however minify-html will automatically add quotes if it's necessary or shorter, so attribute values containing quotes will be safely minified.

**Reply by Mupli** [reply]
Hi, I would like this option too. I'm using template system handlebars and before I use it I minify all templates. 
Just to give example:
Before:
 ...input .. value="**{{someParam}}**" placeholder="something"
After: 
 ...input .. value=**Test value** placeholder=something

(someParam=Test value).

I minify templates long before handling requests so I'm not sure what will be under "someParam" 

I think this issue will kick all people using template systems.
